packages/utils/src/lib/reports/log-stdout-summary.integration.test.ts

```diff
@@ -0,0 +1,47 @@
+import { beforeAll, describe, expect, vi } from 'vitest';
+import { reportMock } from '@code-pushup/test-utils';
+import { ui } from '../logging';
+import { logStdoutSummary } from './log-stdout-summary';
+import { scoreReport } from './scoring';
+import { sortReport } from './sorting';
+
+describe('logStdoutSummary', () => {
+  let logs: string[];
+  beforeAll(() => {
+    logs = [];
+    vi.spyOn(console, 'log').mockImplementation(msg => {
+      logs = [...logs, msg];
+    });
+    // we want to see table and sticker logs in the final style ("raw" don't show borders etc so we use `console.log` here)
```

Comment:
I'm confused. Why is `console.log` mocked and referenced here? I don't see `console.log` being used in the new logging logic.

Reply:
It is used inside the logger that has a renderer that uses either console or an array depending on the mode.

Reply:
that is basically integration testing of the logger

Reply:
added a comment

Reply:
Oh I see, thanks for clarifying!